cmake/FindPulseAudio.cmake

```diff
@@ -35,6 +36,13 @@ find_package_handle_standard_args(PulseAudio
 	VERSION_VAR PULSEAUDIO_VERSION_STRING
 )
 
+if(PULSEAUDIO_LIBRARY AND NOT TARGET PulseAudio::PulseAudio)
```

Comment:
Consider adding that option for Jack

Reply:
??? We don't directly depend on Jack at all